components/Circle.js

```diff
@@ -0,0 +1,55 @@
+import styles from './Circle.module.css';
+
+export default function Circle(props) {
+	const { percent, PercentGap } = props;
```

Comment:
Your naming convention needs to be consistent. If it's `percent`, then it needs to be `percentGap`